### ISSUE TITLE
Throws error for windows full path

### ISSUE THREAD

**Issue by ashantyk** [jduncanator/node-diskusage#29]
This issue only reproduces on Windows OS.

Just giving it the drive letter or, AT MAX, the 3 char string (X:\) the module works just fine, but if you give it a full path (eg.: D:\Temp) the module just spits a NOTFOUND error. 

The module should work with full paths also, just like df works with paths on linux.

**Comment by jduncanator** [comment]
I can't seem to reproduce the issue. What version of Node and Windows are you running? Are you sure you have permission to read the directory you are trying to get disk usage information on?

I've been able to get disk usage information from both the main and external drives, and under subfolders, with and without trailing slashes:

```node
> usage.checkSync('C:\\')
{ available: 200682417152,
  free: 200682417152,
  total: 499084099584 }
> usage.checkSync('C:\\Windows\\')
{ available: 200682417152,
  free: 200682417152,
  total: 499084099584 }
> usage.checkSync('G:\\')
{ available: 240528572416,
  free: 240528572416,
  total: 999835037696 }
> usage.checkSync('G:\\TEMP')
{ available: 240528572416,
  free: 240528572416,
  total: 999835037696 }
> usage.checkSync('G:\\TEMP\\')
{ available: 240528572416,
  free: 240528572416,
  total: 999835037696 }
>
```

**Comment by jduncanator** [comment]
I can't seem to reproduce the issue. What version of Node and Windows are you running? Are you sure you have permission to read the directory you are trying to get disk usage information on?

I've been able to get disk usage information from both the main and external drives, and under subfolders, with and without trailing slashes:

```
> usage.checkSync('C:\\')
{ available: 200682417152,
  free: 200682417152,
  total: 499084099584 }
> usage.checkSync('C:\\Windows\\')
{ available: 200682417152,
  free: 200682417152,
  total: 499084099584 }
> usage.checkSync('G:\\')
{ available: 240528572416,
  free: 240528572416,
  total: 999835037696 }
> usage.checkSync('G:\\TEMP')
{ available: 240528572416,
  free: 240528572416,
  total: 999835037696 }
> usage.checkSync('G:\\TEMP\\')
{ available: 240528572416,
  free: 240528572416,
  total: 999835037696 }
```

**Comment by ashantyk** [comment]
investigated the issue. my bad. it seems i was using **diskspace**, not **diskusage**. sorry
